docs/resources/job.md

```diff
@@ -247,6 +247,11 @@ You can invoke Spark submit tasks only on new clusters. **In the `new_cluster` s
 
 You also need to include a `git_source` block to configure the repository that contains the dbt project.
 
+### run_job_task Configuration Block
+
+* `job_id` - (Required)(String) ID of the job
+* `job_parameters` - (Optional)(Map) Job parameters for the task
```

Comment:
@geekswaroop let's not document Job parameters until the feature goes GA

Reply:
Done